dataprofiler/labelers/data_processing.py

```diff
@@ -1894,8 +1894,8 @@ def __init__(
             random_state = random.Random(random_state)
         elif isinstance(random_state, (list, tuple)) and len(random_state) == 3:
             # tuple required for random state to be set, lists do not work
-            if isinstance(random_state[1], list):
-                random_state[1] = tuple(random_state[1])  # type: ignore
+            if isinstance(random_state, list) and isinstance(random_state[1], list):
+                random_state[1] = tuple(random_state[1])
```

Comment:
RegexPostprocessor and StructCharPostprocessor look like they have a lot of duplicate code. Thoughts on a DRY refactor?

Reply:
In a follow-up PR, open to refactor for this for sure

Reply:
I'd prefer to not hold this PR up if the comments are outside of the diff

Reply:
sounds good!